.github/workflows/bridge.yml

```diff
@@ -0,0 +1,158 @@
+name: tardis-carsus-compatibility-check
+
+on: 
+  workflow_dispatch:
+    inputs:
+      git-ref:
+        description: Git Ref (Optional)    
+        required: false
+        
+env:
+  XUVTOP: /tmp/chianti
+  CHIANTI_DL_URL: https://download.chiantidatabase.org
+  CHIANTI_DB_VER: CHIANTI_v9.0.1_database.tar.gz
+  NBCONVERT_CMD: jupyter nbconvert --execute --ExecutePreprocessor.timeout=600 --to html
+  CMFGEN_DL_URL: http://kookaburra.phyast.pitt.edu/hillier/cmfgen_files
+  CMFGEN_DB_VER: atomic_data_15nov16.tar.gz
+  # original reference data
+  REF1_PATH: /home/runner/work/carsus/carsus/tardis-refdata/unit_test_data_org.h5
```

Comment:
use the variable `${{ github.workspace }}`

Reply:
Done in https://github.com/tardis-sn/carsus/pull/313/commits/42c2bb40df53f0fef1a78755f48aa1176876f73b